.travis.yml

```diff
@@ -2,6 +2,8 @@ script: "bundle exec rake neo4j:install[community-2.3.0] neo4j:start default --t
 language: ruby
 cache: bundler
 sudo: false
+jdk:
+  - oraclejdk8
```

Comment:
Yeah. I was experimenting with errors with previous versions of the neo4j-rb gem and forgot to roll the jdk line back.